### PR TITLE
Update review and deploy tab on V2 create

### PR DESCRIPTION
**Pull request by johnnyd-eth**
## What does this PR do and why?

- Updates V2 create review and deploy to use the 'review and deploy' layout as opposed to project preview
- Adds explanation tooltips to each funding cycle parametre in this section
- Improve mobile styling on V2 create as a whole
- Fixes bug where redemption rate is disabled when distribution limit is Zero (should only be disabled when distribution limit is infinite)

## Screenshots or screen recordings

https://user-images.githubusercontent.com/96150256/166148308-bea5274c-dfba-400d-809d-a8c376392e82.mp4

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).

### CODE REVIEW

File: src/utils/v2/fundingCycle.ts [hunk 1]
```diff
@@ -23,8 +23,7 @@ export const hasDistributionLimit = (
     fundAccessConstraint?.distributionLimit &&
       !parseWad(fundAccessConstraint.distributionLimit).eq(
         MAX_DISTRIBUTION_LIMIT,
-      ) &&
-      fundAccessConstraint.distributionLimit !== '0',
+      ),
```
Comment (johnnyd-eth) [inline src/utils/v2/fundingCycle.ts:26]: Fixes bug where redemption rate is disabled when distribution limit is Zero (should only be disabled when distribution limit is infinite)

File: src/components/v2/V2Project/V2FundingCycleSection/settingExplanations.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { Trans } from '@lingui/macro'
```
Comment (aeolianeth) [inline src/components/v2/V2Project/V2FundingCycleSection/settingExplanations.tsx:1]: **praise** big ups for creating a shared file.

File: src/components/v2/V2Project/V2FundingCycleSection/settingExplanations.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+export const DISCOUNT_RATE_EXPLAINATION = (
```
Comment (aeolianeth) [inline src/components/v2/V2Project/V2FundingCycleSection/settingExplanations.tsx:3]:
```suggestion
export const DISCOUNT_RATE_EXPLANATION = (
```

File: src/components/v2/V2Project/V2FundingCycleSection/settingExplanations.tsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  <Trans>
+    The ratio of tokens rewarded per payment amount will decrease by this
+    percentage with each new funding cycle. A higher discount rate will
+    incentivize supporters to pay your project earlier than later.
+  </Trans>
+)
+
+export const REDEMPTION_RATE_EXPLAINATION = (
```
Comment (aeolianeth) [inline src/components/v2/V2Project/V2FundingCycleSection/settingExplanations.tsx:11]:
```suggestion
export const REDEMPTION_RATE_EXPLANATION = (
```

File: src/components/v2/V2Create/tabs/ReviewDeployTab/index.tsx [hunk 5]
```diff
@@ -3,27 +3,37 @@ import { Trans } from '@lingui/macro'
...
     <div>
-      <h1 style={{ fontSize: '1.5rem' }}>
+      <h1 style={{ fontSize: !isMobile ? '1.5rem' : '1.3rem' }}>
         <Trans>Review project configuration</Trans>
       </h1>
       <div
         style={{
-          padding: '3rem',
+          padding: !isMobile ? '3rem' : '2rem',
           border: '1px solid' + colors.stroke.tertiary,
```
Comment (aeolianeth) [inline src/components/v2/V2Create/tabs/ReviewDeployTab/index.tsx:29]: I think we can just get rid of the padding and border now.

We might still need some bottom margin on the `Review project configuration` h1 tag though - see what you think

**Comment by aeolianeth** [comment]
@johnnyd-eth LGTM, JZ originally wanted the preview here but I agree that the clarity of this view is better.

Couple typos + one UX suggestion: https://github.com/jbx-protocol/juice-interface/pull/895#discussion_r862623893

File: src/components/v2/V2Create/tabs/ReviewDeployTab/index.tsx (hunk 5 above)
Comment (johnnyd-eth) [reply src/components/v2/V2Create/tabs/ReviewDeployTab/index.tsx:29]: Yep done. This how it looks now on desktop/mobile. (Note: still needs a bit of padding on mobile)

Screen Shot 2022-05-02 at 8 12 09 pm

Screen Shot 2022-05-02 at 8 15 44 pm

**Comment by johnnyd-eth** [comment]
@aeolianeth address your latest, thanks! LMK if you're happy and we'll merge

**Review by aeolianeth — APPROVED** [review]
Thanks @johnnyd-eth , merging

**Pull request merged**